cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -373,6 +374,8 @@ protected void configureRoutes() {
         get(recentPath, tsController::getRecent);
         addCacheControl(recentPath, 5, TimeUnit.MINUTES);
 
+        cdaCrudCache("/timeseries/binary/{timeseries}",
```

Comment:
I don't think we discussed it in depth.  Might be worth a revisit.  
Every other controller has the {id} part at the end of the path being registered.  Not saying that is ideal but it is consistent.

Reply:
I could then see wanting some fancy logic to check the Text/Binary parameter so that you don't need the trailing type. The trailing type would be required specifically for annotated versions. 

Kinda muddies up the API though.....